js/src/components/campaign-conversion-notice/index.js

```diff
@@ -1,22 +1,17 @@
 /**
  * External dependencies
  */
-import {
-	Notice,
-	Icon,
-	__experimentalText as Text,
-} from '@wordpress/components';
-import { useState } from '@wordpress/element';
+import { Icon, __experimentalText as Text } from '@wordpress/components';
 import { external as externalIcon } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import { glaData } from '.~/constants';
-import localStorage from '.~/utils/localStorage';
 import AppDocumentationLink from '.~/components/app-documentation-link';
 import CONVERSION_STATUSES from './conversion-statuses';
 import getConversionCampaignStatusNotice from '.~/utils/getConversionCampaignStatusNotice';
+import MigrationCampaignNotice from './migration-notice';
 import './index.scss';
```

Comment:
❔ Question for importing scss: I'm not familiar with css/scss, but we import `index.scss` in this file but did not import it in `campaign-conversion-notice/reports-notice.js`. The styles of the report notice looks great on the browser so it means there is no problem as the styles are loaded by `campaign-conversion-notice/index.js`. I feel like it's not a good way as it's a bit implicit. So maybe we could move the styles of `reports-notice.js` to, for example, `reports-notice.scss` and import it from `reports-notice.js`?

Reply:
Yeah thats a good idea, I have changed in [ebedc71](https://github.com/woocommerce/google-listings-and-ads/pull/1492/commits/ebedc71cd053261e42eb4f2ce3b74b3fe2bdba86)